cluster-operator/src/main/java/io/strimzi/operator/cluster/model/Storage.java

```diff
@@ -15,15 +15,17 @@
 
 /**
  * Represents information about storage to use
+ * @deprecated Replaced with {@link io.strimzi.api.kafka.model.Storage}
  */
+@Deprecated
 public class Storage {
 
     public static final String TYPE_FIELD = "type";
     public static final String SIZE_FIELD = "size";
     public static final String STORAGE_CLASS_FIELD = "class";
     public static final String SELECTOR_FIELD = "selector";
     public static final String SELECTOR_MATCH_LABELS_FIELD = "match-labels";
```

Comment:
The way it worked out I wrote a different, Jackson-annotated, `Storage` class which has a `matchLabels` property. This `Storage` class is now deprecated and I should be able to remove it (in this PR) once I've updated all the testing tests to use the `KafkaAssembly` rather than `ConfigMap`.